core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -352,7 +372,7 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
                    style="background-color: <[collection.thumbnail_bg_color]>;">
               </div>
               <md-card class="oppia-activity-summary-tile top-card">
-                <a ng-click="showCollectionEditor(collection.id)">
+                <a ng-href="/collection_editor/create/<[collection.id]>">
```

Comment:
Hmm.. how about moving this to a similar method as in for explorations - `getCollectionUrl` for consistency ?